scripts/qa_crowbarsetup.sh

```diff
@@ -3271,6 +3272,13 @@ function deploy_single_proposal
                 get_manila_service_instance_details
             fi
             ;;
+        monasca)
+            # PM does not want to support monasca on s390x
+            if [[ $arch = "s390x" ]]; then
```

Comment:
What about aarch64? For now that won't work either, since we've got several packages missing on aarch64: https://build.opensuse.org/project/show/home:jgrassler:monasca